tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -1664,6 +1664,47 @@ public function data_next_tag_ignores_script_tag_contents() {
 		);
 	}
 
+	/**
+	 * Invalid tag names are comments on tag closers.
+	 *
+	 * See https://html.spec.whatwg.org/#parse-error-invalid-first-character-of-tag-name
+	 *
+	 * @ticket 58007
+	 * @dataProvider data_next_tag_ignores_invalid_first_character_of_tag_name_comments
+	 *
+	 * @param string $html_with_markers HTML containing an invalid tag closer whose element before and element after contain the "start" and "end" CSS classes.
```

Comment:
```suggestion
	 * @param string $html_with_markers HTML containing an invalid tag closer whose element before
                                          and element after contain the "start" and "end" CSS classes.
```
Let's break this line to limit the need for horizontal scrolling.

Reply:
broken in ddcb554583.